src/test/java/SalesForceTests/ContractTests.java

```diff
@@ -0,0 +1,29 @@
+package SalesForceTests;
+
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import salesforce.BeforeContract;
+
+public class ContractTests extends BeforeContract {
+
+    @Test(groups = "CreateContract")
+    public void CreateContract(){
+
+        driver.findElement(By.cssSelector(".allTabsArrow")).click();
+        driver.findElement(By.linkText("Contracts")).click();
+        driver.findElement(By.name("new")).click();
+        driver.findElement(By.id("ctrc7")).click();
```

Comment:
Id may change, try to find another way to get the locators, just a recommendation.